<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="Post_Input">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>261</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Publish Post</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="username_stuff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPlainTextEdit" name="username_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="BaconRadio">
         <property name="text">
          <string>Bacon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="follow_radio">
         <property name="text">
          <string>Follow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mute_radio">
         <property name="text">
          <string>Mute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="LikesGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPlainTextEdit" name="post_id_input">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Post ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="like_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="feed_gen_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Feed</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="user_feed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>541</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="usermanagement">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>User management</string>
    </property>
    <widget class="QPlainTextEdit" name="Email_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_create_button">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Create User</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CreateAccount">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>120</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Email_input_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>409</width>
       <height>31</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="username_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>409</width>
       <height>31</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="TerminalOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>541</width>
       <height>271</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
